daemon/environment.cpp

```diff
@@ -109,7 +112,7 @@ static bool exec_and_wait(const char *const argv[])
         // parent
         int status;
 
-        while (waitpid(pid, &status, 0) < 0 && errno == EINTR) {}
```

Comment:
I like the old way better.  They are equivalent, but the {} makes it clear that the loop doing nothing is intentional.

Reply:
done.